Pareto/Checks/Screensaver.swift

```diff
@@ -10,7 +10,7 @@ import os.log
 
 class ScreensaverCheck: ParetoCheck {
     final var ID = "13e4dbf1-f87f-4bd9-8a82-f62044f002f4"
-    final var TITLE = "Screen locks in under 5 minutes"
+    final var TITLE = "Screen saver shows in under 5 minutes"
```

Comment:
Disagree on this one. The point is not that the screen saver shows in under 5 minutes. The point is that the screen is locked. For this, you need two things:
* show screensaver OR go to sleep
* lock screen when screensaver is shown or computer goes to sleep

Reply:
@zupo Unless I'm missing something, the setting for this is literally "show screen saver":

<img width="348" alt="Screenshot 2021-08-02 at 18 06 52" src="https://user-images.githubusercontent.com/1309671/127891274-75551bd0-4ddc-4785-8190-e0a213b8faa4.png">

Reply:
You are missing something. Screen saver is only one way to lock the screen. Sleep is the other.

Reply:
@zupo Please show a screenshot of the setting and where it is.

Reply:
Ah, sorry, here it goes:

<img width="479" alt="Screenshot 2021-08-03 at 10 17 37" src="https://user-images.githubusercontent.com/311580/127982291-1c34448c-70ab-4361-9196-0892bdf3d1e0.png">

Reply:
It's not the same setting:

<img width="819" alt="Screenshot 2021-08-03 at 10 19 51" src="https://user-images.githubusercontent.com/1309671/127982624-f3d2e045-fa65-4fc0-a281-ce750f1e686d.png">

So we either need to add another check, or have this as AND in the current check.

Reply:
This check needs to support both approaches. The end result is that the screen locks in 5 minutes. It does not matter whether through screen saver or sleep.

Reply:
@zupo That is a bug report or new check. As it stands now, the recommended name change is more accurate and reflects the settings better. When we do add the check, we might decide to go with a separate line item, so I would propose we merge this now.

Reply:
The current check is broken. Changing the name will not fix it. Fixing the check will fix it.

Reply:
Energy Saver would be a different check.  Bottom line is that any of [Screen Saver, Security, and Energy Saver] can trigger the "requirement to enter password" unconditionally.  And at the moment we are checking just the screen saver which will trigger more frequently.